src/test/java/items/TestDrawRequestModel.java

```diff
@@ -0,0 +1,25 @@
+package items;
+
+import entities.default_game.IDrawOutputBoundary;
+
+import java.awt.*;
+import java.awt.image.BufferedImage;
+
+
+/**
+ * an IDrawRequestModel implementation for testing only
+ */
+public class TestDrawRequestModel implements IDrawOutputBoundary {
+    @Override
+    public int getTileSize() {
+        return 48;
```

Comment:
I'm guessing you're planning to change this later? The test seems redundant if we know what getTileSize returns.